src/style.css

```diff
@@ -1,3 +1,86 @@
 body {
   margin: 0;
+  padding: 0;
+
+  --active-link-color: #00ACDC;
+}
+
+header {
+  height: 60px;
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+}
+
+ul {
+  height: 100%;
+  display: flex;
+  flex-direction: row;
+  align-items: stretch;
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+li {
+  display: flex;
+  margin-left: 10px;
+  margin-right: 10px;
+}
+
+a {
+  margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
You can add a font family for the parent element to get it inherited by other elements